client/test_commands/deploy_test_token.py

```diff
@@ -2,34 +2,50 @@
 #
 # SPDX-License-Identifier: LGPL-3.0+
 
-from zeth.contracts import Interface
-from zeth.utils import get_zeth_dir
+from zeth.contracts import Interface, send_contract_call
+from zeth.utils import EtherValue, get_zeth_dir, open_web3
 from zeth.constants import SOL_COMPILER_VERSION
-from test_commands.mock import open_test_web3
-from click import command, argument
+from commands.utils import load_eth_address, load_eth_private_key
+from commands.constants import ETH_ADDRESS_DEFAULT
+from click import command, argument, option, pass_context
 from os.path import join
 from solcx import compile_files, set_solc_version
-from typing import Any
-from web3 import Web3  # type: ignore
+from typing import Optional, Any
 
 
 @command()
-@argument("deployer_address")
+@option(
+    "--eth-addr",
+    help=f"Address or address filename (default: {ETH_ADDRESS_DEFAULT})")
+@option("--eth-private-key", help="Sender eth private key")
 @argument("mint_amount", type=int)
 @argument("recipient_address")
+@pass_context
 def deploy_test_token(
-        deployer_address: str,
+        ctx: Any,
+        eth_addr: Optional[str],
+        eth_private_key: Optional[str],
```

Comment:
This would change the flags from `--eth-addr` and `--eth-private-key` to `--eth-addr-file` and `--eth-private-key-file`.  I originally avoided this, as it seems overly verbose.  We could switch to this everywhere - just highlighting the implications of a rename here.

Reply:
Oh that's right. These are the flag names. Ok let's keep it like that, and make sure we mention - in the flag documentation - that this is a filename (as you did below)